blocks/table/table.css

```diff
@@ -0,0 +1,134 @@
+/* stylelint-disable no-descending-specificity */
+.table-container {
+    margin-bottom: 32px;
+}
+
+.table-container > div.table-wrapper {
+    margin-bottom: 0;
+}
+
+.table-container > .table-wrapper + .table-wrapper {
+    margin-top: 0;
+}
+
+.table-form-submit-wrapper {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.table-form-submit-wrapper button {
+    height: 50px;
+    width: 300px;
+    background-color: var(--blue);
+    color: var(--white);
+    border-radius: 6px;
+    font-family: var(--heading-font-family);
+    font-size: var(--heading-size-s);
+    margin: 20px 0;
+}
+
+.table-form-submit-wrapper button:disabled {
+    background-color: var(--blue-light);
+}
+
+.table-wrapper:not(:has(+ .table-wrapper)) .table tbody tr:last-of-type td {
+    border-bottom: 2px solid var(--blue);
+}
+
+.table {
+    width: 100vw;
+    margin-left: calc(-50vw + 50%);
+}
+
+.table:first-of-type {
+    border-top: 2px solid var(--blue);
+}
+
+.table table {
+    border-spacing: 0;
+    width: 100%;
+}
+
+.table thead th {
+    padding: 0 12px;
+    border-bottom: 2px solid var(--blue);
+    font-family: var(--heading-font-family);
+    font-size: var(--heading-size-xs);
+    text-transform: uppercase;
+    background-color: var(--blue-lightest);
+}
+
+.table thead th:first-child {
+    text-align: left;
+    border-right: 2px solid var(--blue);
+}
+
+.table thead th:last-child {
+    width: 125px;
+}
+
+.table tbody tr td {
+    padding: 16px 12px;
+    border-bottom: 2px solid var(--blue);
+}
+
+.table tbody tr:last-child td {
+    border-bottom: none;
+}
+
+.table tbody tr td:first-child {
+    border-right: 2px solid var(--blue);
+    position: relative;
+}
+
+.table table tbody tr td:first-child p:not([class]) {
+    margin-right: 30px;
+    margin-bottom: 0;
+    font-size: var(--body-size-s);
+}
```

Comment:
this is relying on there only being one paragraph of descriptive text, correct?

Reply:
Fixed this one too, I made the selector more specific. `.table table tbody tr td:first-child p:not([class]):nth-child(2)`